values.yaml

```diff
@@ -49,6 +49,8 @@ web:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  ingress:
+    enabled: false
 
```

Comment:
values should also define an empty list of hosts so it is documented.

Reply:
added